packages/p2p/src/components/order-details/order-details.jsx

```diff
@@ -92,6 +92,11 @@ const OrderDetails = observer(() => {
             disposeReactions();
             order_store.setOrderPaymentMethodDetails(undefined);
             order_store.setOrderId(null);
+            general_store.props.setP2POrderProps({
+                order_id: order_store.order_id,
+                redirectToOrderDetails: general_store.redirectToOrderDetails,
+                setIsRatingModalOpen: order_store.setIsRatingModalOpen,
+            });
```

Comment:
set the p2p_order_props when unmounting order-details page as if the user navigates from the order-details page to another page outside of p2p, then if they click the give feedback button in notifications that has the same order_id as the previous order-details page they had just left, they would not be able to navigate to the same order page. Hence the need to reset the p2p_order_props.